cg/apps/demultiplex/sample_sheet/novaseq_sample_sheet.py

```diff
@@ -88,22 +96,37 @@ def convert_sample_to_header_dict(
         sample_dict = sample.dict(by_alias=True)
         return [str(sample_dict[header]) for header in sample_sheet_headers]
 
+    def create_v2_sample_sheet_sections(self) -> List[str]:
+        """Create the header and read sections of a v2 sample sheet in a list."""
+        return [
```

Comment:
Should we create this via a pydantic module instead. We could create a base module for sheets and inherit from it for different type of sheets. Then we can also validate when creating